src/main/java/school/lemon/changerequest/java/introduction/hw2/Task1.java

```diff
@@ -2,6 +2,10 @@
 
 public class Task1 {
     public static String makeTags(String tag, String text) {
-        return "";
+        String opentag = "<" + tag + ">";
+        String closetag = "</" + tag + ">";
+        String html = opentag + text + closetag;
+
+        return html;
```

Comment:
It's OK, but IMO, it's better to solve this task with `StringBuilder` and the best solution will be using `String.format()`.